### ISSUE TITLE
hie-bios targets rely on the `bazel-out` symlink

### ISSUE THREAD

**Issue by ylecornec** [tweag/rules_haskell#1857]
`hie-bios` targets used for ide integration output paths that are relative to the workspace root, and rely on the existence of the `bazel-out` symlink, which is not there in bazel 5 if using the `--symlink_prefix` option. 

This is likely the reason behind [this ci error](https://github.com/tweag/rules_haskell/actions/runs/3853503897/jobs/6566546189#step:7:138).

**Comment by avdv** [comment]
Here's a workaround which might be useful:
```diff
diff --git a/.hie-bios b/.hie-bios
index 31f3736a..e65326ee 100755
--- a/.hie-bios
+++ b/.hie-bios
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 hie_bios_flags() {
+  local -x OUTPUT_PATH=$( bazel info output_path )
   bazel build //tests:hie-bios \
     --output_groups=hie_bios \
     --experimental_show_artifacts \
@@ -8,6 +9,7 @@ hie_bios_flags() {
     | awk '
       /^>>>/ {
         while ((getline line < substr($1, 4)) > 0) {
+          sub(/^bazel-out\//, ENVIRON["OUTPUT_PATH"] "/", line)
           print line
         }
         next
```

**Comment by ylecornec** [comment]
Closing as fixed by #1862
